sdk/nodejs/Makefile

```diff
@@ -123,14 +125,16 @@ TEST_GREP ?= .*
 test_watch::
 	PULUMI_TEST_ORG=$(PULUMI_TEST_ORG) yarn mocha "**/*.spec.ts" --timeout 300000 --bail -j 1 --watch --watch-files "**/*.ts" --grep "$(TEST_GREP)"
 
-dist:: build
-	cp dist/pulumi-resource-pulumi-nodejs "$$(go env GOPATH)"/bin/
-	cp dist/pulumi-analyzer-policy "$$(go env GOPATH)"/bin/
+dist::
+	GOBIN=${GOBIN} go install -C cmd/pulumi-language-nodejs \
+		-ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ${LANGHOST_PKG}
```

Comment:
Hrm why can't `dist` not still rely on `build` to build the binaries?  Would be nice if we could just re-use the binaries from that.

Reply:
We probably can, I copied the Python makefile and didn't want to have too many changes here and kept things dumb.